src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifierDescriptor.java

```diff
@@ -47,22 +47,12 @@
 public final class PhabricatorNotifierDescriptor extends BuildStepDescriptor<Publisher> {
     private String credentialsID;
     private String uberallsURL;
-    private String commentFile;
-    private String commentSize;
 
     public PhabricatorNotifierDescriptor() {
         super(PhabricatorNotifier.class);
         load();
     }
 
-    public String getCommentSize() {
-        return commentSize;
-    }
-
-    public void setCommentSize(String commentSize) {
-        this.commentSize = commentSize;
-    }
```

Comment:
I need to triple-check, but it appears that these fields and getter/setters were added by mistake, and were only called in the test suite -- the descriptor is used for "global" config like uberalls URL and conduit credentials, not job-specific stuff.